customer_satisfaction/README.md

```diff
@@ -0,0 +1,90 @@
+
+The purpose of this repository is to demonstrate how [ZenML](https://github.com/zenml-io/zenml) empowers your business to build and deploy machine learning pipelines in a multitude of ways:
+
+* By offering you a framework or template to develop within
+* By integration of tools like MLflow deployment, MLflow tracking and more
+* By allowing you to build and deploy your machine learning pipelines easily 
+
+## :snake: Python Requirements
+
+Let's jump into the Python packages you need. Within the Python environment of your choice, run:
+
+```bash
+git clone https://github.com/zenml-io/zenfiles.git
+cd customer_satisfaction
+pip install -r requirements.txt
+```
+
+If you are running the `run_deployment.py` script, you will also need to install some integrations using ZenML:
+
+```bash
+zenml integration install mlflow -f
+```
+
+## :thumbsup: The Task
+
+Our team at ZenML was looking for a project which is model-centric, data-centric and ZenML centric for our next Zenfile. During our initial discussions, we listed several questions like What will be the use case for this zenfile? What's the impact of this zenfile? etc., to evaluate a zenfile. We found out that it would be great if we build an end-to-end pipeline for predicting the customer satisfaction score for the next order or purchase, which will help businesses make better decisions. We can't just train our model in our local system, and we need to serve the users, so we need it to be deployed in the cloud. We need machine learning pipelines for machine learning at scale, an end-to-end construct that orchestrates the flow of data into and output from a machine learning model (or set of multiple models). It includes raw data input, features, results, the machine learning model and model parameters, and prediction outputs. And All these capabilities are built on top of the ZenML framework.
+
+As a result of this discussion, we decided to build a pipeline to predict the customer satisfaction score for the next order or purchase. We gave special consideration to ZenML standard pipeline and its steps, MLflow tracking to track our metrics and parameters, and MLflow deployment to deploy our model. 
+
+Our standard training pipeline consists of several steps: 
+
+* ingest_data:- This step will ingest the data from the dataset and create a data frame.
+* clean_data:- This step will clean the data and remove the unwanted columns. 
+* model_train:- This step will train the model and save the model using MLflow auto logging. 
+* evaluation:- This step will evaluate the model and save the metrics using mlfow autlogging into the artifact store.  
+
+### Depoloyment & Inference Pipeline 
+
+We have another pipeline, the deployment_pipeline.py, that implements a continuous deployment workflow. It ingests and processes input data, trains a model and then (re)deploys the prediction server that serves the model if it meets some evaluation criteria.
```

Comment:
Maybe, instead of just saying 'some evaluation criteria', you could state what the criteria is?